statgen/fivex
# Feature/credible set and single variant updates

## Ticket
#71

## Purpose
To add short gene names to the table on the bottom of the region view page. To accommodate this, we reprocessed credible_sets files, adding an extra column at the very end for short gene names for the merged (from multiple single-study files) and joined (to add p-values and other extra information) credible_sets data. We also changed the corresponding region view table formats to add these short gene names to the table.
Includes a fix for a typo in the region view table (previously a column was labeled -log10(p-value), but the values were actually p-values; the values are now properly transformed to -log10(p-value), matching that in the single-variant view).

## How to test this
Look at single-variant and region views to make sure the pages aren't broken
Check the table to make sure the values are correct

## Deployment / configuration notes
Need to replace old credible_sets files with updated files on the production server before deployment
